cities/management/commands/cities.py

```diff
@@ -534,19 +533,25 @@ def import_district(self):
 
         self.logger.info("Building city index")
         city_index = {}
-        for obj in City.objects.all():
+        for obj in tqdm(City.objects.iterator(),
+                         total=City.objects.count(),
+                         desc="Building City index"):
             city_index[obj.id] = obj
 
         self.logger.info("Importing district data")
         for item in tqdm(data, total=total, desc="Importing districts"):
             if not self.call_hook('district_pre', item):
                 continue
 
-            type = item['featureCode']
-            if type not in district_types:
+            item_type = item['featureCode']
+            if item_type not in district_types:
                 continue
 
             district = District()
+            try:
+                district.id = int(item['geonameid'])
```

Comment:
What about for users who have already imported districts? Those districts will have their own ID, not the `item['geonameid']`.

@coderholic Your thoughts?